app/routes/helpers/yes-no-radios.js

```diff
@@ -5,12 +5,12 @@
  * @param {string} id of the radio component.
  * @param {string} previousAnswer either `yes` or `no`, used to set the radio as checked.
  * @param {string} [errorText] error message to display.
- * @param {object} [options] an object containing `isPageHeading` (boolean, determines if the legendText is marked as a heading), `legendClasses` (string, contains classes to add to legendText) and `inline` (boolean determines if the class to make the radios display as inline is added).
+ * @param {object} [options] an object containing `isPageHeading` (boolean, determines if the legendText is marked as a heading), `legendClasses` (string, contains classes to add to legendText), `inline` (boolean determines if the class to make the radios display as inline is added) and hintText (string to display hint message with the radio button).
  *
  * @return {object} object with `radios` property containing radios component.
  */
-function getYesNoRadios (legendText, id, previousAnswer, errorText = undefined, options = { isPageHeading: true, legendClasses: 'govuk-fieldset__legend--l', inline: true }) {
-  const { inline, isPageHeading, legendClasses } = options
+function getYesNoRadios (legendText, id, previousAnswer, errorText = undefined, options = { isPageHeading: true, legendClasses: 'govuk-fieldset__legend--l', inline: true, hintText: '' }) {
+  const { inline, isPageHeading, legendClasses, hintText } = options
```

Comment:
If `options` is included without all necessary properties they aren't going to be populated and will change the behaviour. The code snippet in the last comment is going to be required. It would be good to have a test for this too please.